### DISCUSSION TITLE
Github messed up my account!

### DISCUSSION THREAD

**Discussion by Nusab19** [community/community#44543] [New to GitHub] [answered]
Hello!

My emails have been untracked!

I created this account in github with `x` email.

And I had another github account with `nusab_____@gmail.com`.

As @Nusab19 was my main account, so I *changed* the email from `x` to `nusab_____@gmail.com`.

But I forgot that I already had an account associated with that(`nusab_____@gmail.com`) gmail.

So, whenever I pushed any commit to any public repository, my other account's Username was shown.

So, I deleted that account. But somehow, Now if I push any commit, I shows some weird Username `= and =` (see in the image)

I am using an Classic Auth-Token in Termux.

It'd be helpful if Github fixes this issue so whenever I push to any repository, it shows "Nusab19" instead of "= and ="

Thanks for reading this unorganised text.

<img src="https://te.legra.ph/file/34062f5b41c4aa587234b.jpg">

**Comment by chris-c-thomas** [comment, marked as answer]
You can [merge two accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts). Probably what you wanted to follow from the get go but hopefully we can recover this to what you're looking for.

But I think now what you'll want to do is just make sure all the associated emails are on the current account that you're using. The commit history should get linked to @Nusab19 - try going to https://github.com/settings/emails and adding them all. So that fixes your GitHub profile. Hopefully.

Next you'll want to configure git to use the email you want for each repository. You can do this local to each repo or set a global and have that be what you use / fallback option if there's no local config set.

From within your repo run `git config --list` to see what is set for the repository. Change it as desired so that when you push from the command line it uses the email you want.

**Comment by DevKren** [comment]
My account got marked as flagged and I have a career fair tomorrow and I am worried that recruiters for internships will not be able to see my GitHub account because it has been flagged.
